src/StreamJsonRpc/LengthHeaderMessageHandler.cs

```diff
@@ -109,7 +110,7 @@ protected override void Write(JsonRpcMessage content, CancellationToken cancella
             }
 
             // Now go back and fill in the header with the actual content length.
-            Utilities.Write(this.Writer.GetSpan(sizeof(int)), checked((int)contentSequence.Length));
+            BinaryPrimitives.WriteInt32BigEndian(this.Writer.GetSpan(sizeof(int)), checked((int)contentSequence.Length));
```

Comment:
Is this another one that's going to cause you to load a library you didn't want to?

Reply:
Thanks for checking. Ya, we're using System.Memory heavily throughout the library.